posthog/models/performance/sql.py

```diff
@@ -92,6 +90,8 @@
 navigation_type LowCardinality(String),
 unload_event_end Float64,
 unload_event_start Float64,
```

Comment:
I think every number column here should technically be `nullable<blah>` at least one of them uses absent vs 0 to mean different things

since we're dropping the table anyway we can make this change now easily